### DISCUSSION TITLE
Unable to run the Sophia script on Windows 20H2 (OS Build 19042,954)

### DISCUSSION THREAD

**Discussion by lmfodor** [farag2/Sophia-Script-for-Windows#172] [General]
Hi! I downloaded the latest version that indicate that support 2021 May Update, however when I run the script I received the following error

The script supports Windows 10 1809 Enterprise LTSC.

What I am doing wrong? Or this version is not supported yet?

Thanks!

**Comment by farag2** [comment]
What the archive did you download?

**Reply by farag2** [reply]
😞
